core/src/test/java/org/elasticsearch/test/rest/ElasticsearchRestTestCase.java

```diff
@@ -295,10 +295,7 @@ private static void validateSpec(RestSpec restSpec) {
 
     @AfterClass
     public static void close() {
-        if (restTestExecutionContext != null) {
-            restTestExecutionContext.close();
-            restTestExecutionContext = null;
```

Comment:
there is no need to close connections anymore? The previous http client did some connection pooling too, added here https://github.com/elastic/elasticsearch/commit/7776b6b4f0fc105d9261c1140c683c70e7fc0979 . How does this client work in terms of connections?